examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImWriteCommand.kt

```diff
@@ -0,0 +1,111 @@
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.controller.commands.pairing
+
+import chip.devicecontroller.ChipDeviceController
+import chip.devicecontroller.GetConnectedDeviceCallbackJni.GetConnectedDeviceCallback
+import chip.devicecontroller.WriteAttributesCallback
+import chip.devicecontroller.model.AttributeWriteRequest
+import chip.devicecontroller.model.ChipAttributePath
+import com.matter.controller.commands.common.CredentialsIssuer
+import java.util.ArrayList
+import java.util.logging.Level
+import java.util.logging.Logger
+import javax.annotation.Nullable
+
+private const val MATTER_PORT = 5540
+private const val CLUSTER_ID_BASIC = 0x0028L
+private const val ATTR_ID_LOCAL_CONFIG_DISABLED = 16L
+
+class PairOnNetworkLongImWriteCommand(
+  controller: ChipDeviceController, credsIssue: CredentialsIssuer?
+) : PairingCommand(
+  controller,
+  "onnetwork-long-im-write",
+  PairingModeType.ON_NETWORK,
+  PairingNetworkType.NONE,
+  credsIssue,
+  DiscoveryFilterType.LONG_DISCRIMINATOR
+) {
+  private var devicePointer: Long = 0
+  private fun setDevicePointer(devicePointer: Long) {
+    this.devicePointer = devicePointer
+  }
+
+  private inner class InternalWriteAttributesCallback : WriteAttributesCallback {
+    override fun onError(@Nullable attributePath: ChipAttributePath?, e: Exception?) {
+      logger.log(Level.INFO, "Write receive onError on ")
+      if (attributePath != null) {
+        logger.log(Level.INFO, attributePath.toString())
+      }
+      setFailure("write failure")
+    }
+
+    override fun onResponse(attributePath: ChipAttributePath?) {
+      logger.log(Level.INFO, "Write receve OnResponse on ")
+      if (attributePath != null) {
+        logger.log(Level.INFO, attributePath.toString())
+      }
+      setSuccess()
+    }
+  }
+
+  private inner class InternalGetConnectedDeviceCallback : GetConnectedDeviceCallback {
+    override fun onDeviceConnected(devicePointer: Long) {
+      setDevicePointer(devicePointer)
+      logger.log(Level.INFO, "onDeviceConnected")
+    }
+
+    override fun onConnectionFailure(nodeId: Long, error: Exception?) {
+      logger.log(Level.INFO, "onConnectionFailure")
+    }
+  }
+
+  override fun runCommand() {
+    // boolean true for tlv
+    val booleanTLV = byteArrayOf(0x09)
+    val attribute: AttributeWriteRequest = AttributeWriteRequest.newInstance( /* endpointId= */
+      0, CLUSTER_ID_BASIC, ATTR_ID_LOCAL_CONFIG_DISABLED, booleanTLV
+    )
+    val attributeList: ArrayList<AttributeWriteRequest> = ArrayList<AttributeWriteRequest>()
```

Comment:
This can be simplified now:

```
val attributeList = listOf(
  AttributeWriteRequest.newInstance(
    /* endpointId= */ 0,
    CLUSTER_ID_BASIC,
    ATTR_ID_LOCAL_CONFIG_DISABLED,
    booleanTLV,
  )
)
```